### ISSUE TITLE
Suggestion for added visibility in tab list [BETA]

### ISSUE THREAD

**Issue by magik3r** [mbnuqw/sidebery#1153]
## Have two (or more) panels in view at once
This might be a silly suggestion but I think it could be useful to be able to open several panels simultaneously for cases where you might want to be structured i.e frequently moving tabs between panels or simply just want easier visibility when switching between multiple tabs. 

Example behavior could be that you drag the icon of a panel (not currently active) into the tab list of the current panel splitting it to show both panels (and their contents).

Maybe this is already possible through style editor / userChrome.css but I am not familiar enough with either to grasp their respective capabilities - Apologies in advance if this is already doable or if I'm overlooking any similar functionality.

EDIT: An other solution might be to allow pinned groups to still be expandable like they normally are or allowing the ability to create expandable folders etc that remain after closing firefox.

**Comment by mbnuqw** [comment]
> Maybe this is already possible through style editor

No, sidebery internals only work with a single `activePanelId`, so CSS-only solution won't be enough.


> An other solution might be to allow pinned groups to still be expandable like they normally are or allowing the ability to create expandable folders etc that remain after closing firefox.

Not sure I got it right, could you describe it in more details?

**Comment by magik3r** [comment]
> > An other solution might be to allow pinned groups to still be expandable like they normally are or allowing the ability to create expandable folders etc that remain after closing firefox.
> 
> Not sure I got it right, could you describe it in more details?

Here is an example of two groups exhibiting different behavior depending on whether they are pinned or not (both contain tabs):

EDIT: I realized that this solution would also require:
- Pinned groups to be expandable.
-  The option to allow pinned tabs to be moved anywhere in the order of tabs within a panel (rather than the current options of either having pinned tabs at the top of a panel or globally visible).
- (Allow group icons to be edited)
![pinned_group](https://github.com/mbnuqw/sidebery/assets/64514975/54e53b91-47cb-4cab-b3ec-b4199e87f07b)



> This might be a silly suggestion but I think it could be useful to be able to open several panels simultaneously for cases where you might want to be structured i.e frequently moving tabs between panels or simply just want easier visibility when switching between multiple tabs.

Just for added clarity i put together an example in paint of what it would look like to have two panels open in "split" view:
**Apologies for not providing this in the initial post.**
![split_panel_example](https://github.com/mbnuqw/sidebery/assets/64514975/e26f1541-6d9d-4749-ab28-4c668f6d880d)

**NOTE:** I personally think that the second example looks more organized and useful seeing as panels already have useful functions such behavior when opening new tabs etc. as well as settings like managing rules and changing icons.

**Comment by mbnuqw** [comment]
Thanks for explanation, now I understand what you meant.

- Pinned groups (and in general preserving tabs tree in pinned tabs)
  It's a `wontfix` because it doesn't work well in inline layout (when a pinned tab is an icon-only element).

- The option to allow pinned tabs to be moved anywhere in the order of tabs within a panel
  It's impossible because Firefox is not allowing it and Sidebery directly represents native tabs in their original order so that the native keybindings for switching/moving tabs work properly.

- Allow group icons to be edited
  :+1: It's planned.

- Split view
  :+1: It's also planned (I think there was an issue for this, but I can't find it...)
